src/main/kotlin/uk/gov/justice/digital/hmpps/adjustments/api/service/AdjustmentsService.kt

```diff
@@ -105,18 +106,9 @@ class AdjustmentsService(
       }!!
     return mapToDto(adjustment)
   }
-
-  fun findByPerson(person: String): List<AdjustmentDto> {
-    return adjustmentRepository.findByPerson(person)
-      .filter { !it.deleted }
-      .map { mapToDto(it) }
-  }
-
```

Comment:
The below `findByPersonAndSource` functionality was removed as it is not used by anything. Removed so don't have to maintain the code